assets/javascripts/modules/mma/updatePayment.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import ReactDOM from 'react-dom';
+
+
+
+export const init = (elem) => {
+    let handler = window.StripeCheckout.configure(guardian.stripeCheckout)
+    ReactDOM.render(<Payment sub={elem.dataset.subId} email={elem.dataset.email} stripe={handler} />, elem)
+}
+
+const getDetails = async (url) => {
+    console.log(url)
+    let resp = await fetch(`${url}/user-attributes/me/mma-paper`, { method: 'get', credentials: 'include' })
+    // let json = await resp.json()
+    let json = await resp.json()
+    console.log(resp, json)
+    return json
+}
+const WAITING = 'waiting'
+const SUCCESS = 'success'
+const FAILURE = 'failure'
+const FORM = 'form'
+
+const CardUpdate = ({ card, handler }) => (<div><button className="button button--primary button--large" onClick={handler}>•••• •••• •••• {card.last4} Update Card</button></div>)
+const Success = () => (<p>Thank you, we have successfully updated your payment details.</p>)
+const Failure = () => (<p>Unfortunately, we are unable to update your payment details at this time, please contact the call centre.</p>)
```

Comment:
We should provide contact details here

Reply:
Yeah, I'm still waiting on approved copy here.